allennlp/training/metrics/metric.py

```diff
@@ -34,6 +38,26 @@ def get_metric(
         """
         raise NotImplementedError
 
+    def get_metric_name_value_pairs(
```

Comment:
This function is necessary to have a homogeneous output out of `get_metric`. It can be here, or it could be a util function maybe.

Reply:
Another option is to change `get_metric`.